.codesandbox/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM node:20-slim
+
+RUN corepack enable && corepack prepare pnpm@$(yq e '.engines.pnpm' package.json)
```

Comment:
make sure there's a line at the end of the file - eg. this "no empty line at end" symbol should not be here:

![Screenshot 2024-05-13 at 12 22 37](https://github.com/upleveled/express-guest-list-api-memory-data-store/assets/1935696/26ae8a84-7342-44fc-ad27-5d1b2d215b53)

Reply:
Added empty line in this commit [c32a304](https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/294/commits/c32a304b0e15ad240375c8771374fbabb1d855e6)